packages/portal/spec-renderer-mini/sandbox/App.vue

```diff
@@ -65,11 +61,11 @@
 
 <script setup lang="ts">
 import { onMounted, ref, watch } from 'vue'
-import { SpecItemType, SpecTag } from '../src/types'
+import { Operation, Tag } from '../src/types'
 import { KButton, KLabel } from '@kong/kongponents'
 import SpecRendererMini from '../src'
```

Comment:
Don't we need to update the name of this folder (and file) if we aren't calling it spec renderer mini anymore?